src/mobile-ui-react/ResizablePanel.tsx

```diff
@@ -414,6 +418,48 @@ class TouchCaptor extends React.PureComponent<TouchCaptorProps> {
   };
 }
 
+// @todo Use the functional version of TouchCaptor below after sufficient testing.
```

Comment:
Maybe add a note and link about the usePointerCaptor function as a possible alternative

Reply:
I did so after the other todo comment.